packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -23,7 +23,7 @@
   "categories": ["Languages"],
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "41.18.0",
-    "lwc-language-server": "0.14.2",
+    "lwc-language-server": "0.15.1",
```

Comment:
@rsalvador - Just a quick sanity check. Since this is for the 212 Closed Pilot, I should use 0.15.1 here instead of 1.0.1, right?

Reply:
right

Reply:
we will make a new lwc-language-version for 212 tomorrow, I don't know if you want to wait for it or make another release later

Reply:
I will wait and then incorporate that new one tomorrow.